### PR TITLE
Scene

### PR DESCRIPTION
**Pull request by kkucharc**
Added menu with wires. Animation. Added background balls.

### CODE REVIEW

File: src/components/elements/neon.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            <NeonPlane id="neon-plane"/>
+            <NeonOutline id="neon-outline"/>
+            <NeonText id="neon-text"/>
+        </Entity>
+    }
+}
+
+class NeonPlane extends React.Component {
+    render() {
+        return <Entity {...this.props} ply-model="src: #neonPlane"/>
```
Comment (KrzysztofWrobel) [inline src/components/elements/neon.js]: Better to have another entity around it

File: src/components/elements/windmill.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Entity} from "aframe-react";
 
 
-export default class Windmill extends React.Component {
```
Comment (KrzysztofWrobel) [inline src/components/elements/windmill.js]: Why default was removed?

File: src/components/elements/wires.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            <WireTurn id="wire-turn"/>
+            <WireVertical id="wire-vertical"/>
+        </Entity>
+    }
+}
+
+class WireFork extends React.Component {
+    render() {
+        return <Entity className="item intersectable" hoverable hovered_menu_item>
+            <Entity className="item intersectable" scale={{x: 0.05, y: 0.05, z: 0.05}}
```
Comment (KrzysztofWrobel) [inline src/components/elements/wires.js]: Remove intersactable on bottom element

File: src/components/elements/wires.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    position={{x: -0.6, y: 0.2, z: -0.5}} rotation={{x: 0, y: 0, z: 0}} ply-model="src: #wireFork"/>
+        </Entity>
+    }
+}
+
+class WireHorizontal extends React.Component {
+    render() {
+        return <Entity className="item intersectable" hoverable hovered_menu_item>
+            <Entity className="item intersectable" scale={{x: 0.05, y: 0.05, z: 0.05}}
```
Comment (KrzysztofWrobel) [inline src/components/elements/wires.js]: As above

File: src/components/elements/wires.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            <WireVertical id="wire-vertical"/>
+        </Entity>
+    }
+}
+
+class WireFork extends React.Component {
+    render() {
+        return <Entity className="item intersectable" hoverable hovered_menu_item>
+            <Entity className="item intersectable" scale={{x: 0.05, y: 0.05, z: 0.05}}
+                    position={{x: -0.6, y: 0.2, z: -0.5}} rotation={{x: 0, y: 0, z: 0}} ply-model="src: #wireFork"/>
```
Comment (KrzysztofWrobel) [inline src/components/elements/wires.js]: Remove default rotation on 0, 0, 0

File: src/components/elements/wires.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                    position={{x: 0.6, y: 0.2, z: -0.5}} rotation={{x: 0, y: 0, z: 0}}
+                    ply-model="src: #wireHorizontal"/>
+        </Entity>
+    }
+}
+
+class WireTurn extends React.Component {
+    render() {
+        return <Entity className="item intersectable" hoverable hovered_menu_item>
+            <Entity className="item intersectable" scale={{x: 0.05, y: 0.05, z: 0.05}}
```
Comment (KrzysztofWrobel) [inline src/components/elements/wires.js]: As  above

File: src/components/elements/wires.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+                    position={{x: -0.6, y: -1, z: -0.5}} rotation={{x: 0, y: 0, z: 0}} ply-model="src: #wireTurn"/>
+        </Entity>
+    }
+}
+
+class WireVertical extends React.Component {
+    render() {
+        return <Entity className="item intersectable" hoverable hovered_menu_item>
+            <Entity
+                scale={{x: 0.05, y: 0.05, z: 0.05}} position={{x: 0.6, y: -1, z: -0.5}} rotation={{x: 0, y: 0, z: 0}}
```
Comment (KrzysztofWrobel) [inline src/components/elements/wires.js]: Remove default rotation

File: src/index.js [hunk 8]
```diff
@@ -1,23 +1,30 @@
...
+
+import {Scene} from "aframe-react";
 import React from "react";
 import ReactDOM from "react-dom";
 import View from "./components/view";
 import Assets from "./components/assets";
 import Controlers from "./components/controlers";
 import "./components/properties/hoverable";
+// Register a single component.
+extras.physics.registerAll();                               // Register a particular package, and its dependencies.
```
Comment (KrzysztofWrobel) [inline src/index.js]: Is it needed to registerAll? How about only those we use?

**Review by KrzysztofWrobel — CHANGES_REQUESTED** [review]

File: src/components/elements/windmill.js (hunk 2 above)
Comment (kkucharc) [reply src/components/elements/windmill.js]: I added there other models, then I removed them. I will bring it back.

**Pull request merged**